### ISSUE TITLE
批量归一化 batch normalization - 会飞的咸鱼

### ISSUE THREAD

**Issue by TankZhouFirst** [TankZhouFirst/TankZhouFirst.github.io#7]
https://tankzhou.cn/2019/07/01/%E6%89%B9%E9%87%8F%E5%BD%92%E4%B8%80%E5%8C%96-Batch-Normalization/

**Comment by TankZhouFirst** [comment]
批量归一化 batch normalization
